dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/CDIUpgradeHandlerTest.java

```diff
@@ -248,30 +231,24 @@ public void implTestCDINoUpgrade() throws Exception {
         String testName = "testCDINoUpgrade";
 
         logStart(methodName, testName);
-
-        WebBrowser browserS1R1 = createWebBrowser();
-        browserS1R1.setAutoRedirect(false);
-        browserS1R1.setAcceptCookies(true);
-
+        
+        HttpClientParams params = new HttpClientParams();
+        params.setCookiePolicy(CookiePolicy.DEFAULT);
```

Comment:
What is the default cookie policy? Does it need to be set?

Reply:
It just makes sure cookies are enabled.